src/protocol/thread/thread.ts

```diff
@@ -928,6 +928,8 @@ class _ThreadFront {
     onConsoleOverflow: () => void
   ) {
     const sessionId = await this.waitForSession();
+    // Wait for basic processing to start before fetching console messages.
+    await this.ensureProcessed("basic");
```

Comment:
This needs to be passed the `sessionId` :P

Reply:
Does it?? https://github.com/RecordReplay/devtools/blob/9e8035861d1f30e4ef64d33dd8f3feada26b0e68/src/protocol/thread/thread.ts#L274-L276